libcxx.mixin

```diff
@@ -0,0 +1,8 @@
+build:
+  libcxx:
```

Comment:
Maybe the mixin name should reflect the fact that not only `libc++` is being used but also `clang`.

Reply:
I agree, updated to be called `clang-libcxx`